cdci_data_analysis/configurer.py

```diff
@@ -33,89 +33,62 @@
 
 logger = logging.getLogger("conf")
 
-
-# TODO: this looks rather specific to INTEGRAL?
-
 class DataServerConf:
 
-    def __init__(self, data_server_url,
-                 data_server_remote_cache=None,
-                 dispatcher_mnt_point=None,
-                 dummy_cache=None,
```

Comment:
dummy_cache is not entirely optional. also products_url. they are also used by all instruments.

It is true that current implementation would allow it anyway, and fail in dummy request, and make a problem with product url.
so we rather want them to be always set in production, but it might be useful to allow it to be None in test (or in current Kyiv deployment).  

But this situation should at least show a warning, or ideally raise a problem if server is not started in some "debug" mode (the same mode that reports extra details in run_analysis issues)

 data_server_remote_cache and dispatcher_mnt_point are the only two INTEGRAL specific here.